### PR TITLE
Improve Select2 compatibility with other drivers

### PR DESCRIPTION
**Pull request by craig410**

### CODE REVIEW

File: src/Javascript/Control/Select2v4.php [hunk 1]
```diff
@@ -40,12 +39,10 @@ class Select2v4
 
     protected function __construct(Session $session, NodeElement $base_select)
     {
-        if ( ! $session->getDriver() instanceof Selenium2Driver) {
-            throw new UnsupportedDriverActionException(
-                __CLASS__.' needs a selenium driver, got %s',
-                $session->getDriver()
-            );
-        }
+        //guard against drivers which don't support JS
```
Comment (acoulton) [inline src/Javascript/Control/Select2v4.php]: Maybe worth extracting this to a static method of it's own - `Ingenerator\BehatSupport\Mink\BrowserChecker::requireJavascript()` or something? 

We don't use it anywhere else here but could reuse it in the projects, then the intent is clear and we can rejig exactly how we check / what we accept in future if necessary?

**Review by acoulton — CHANGES_REQUESTED** [review]
All looks good bar maybe extracting a method, nice one

File: src/Javascript/Control/Select2v4.php (hunk 1 above)
Comment (craig410) [reply src/Javascript/Control/Select2v4.php]: Yeah that's a good plan

**Review by acoulton — COMMENTED** [review]
LGTM have a rollout rat :) :shipit:

**Pull request merged**